examples/ERP/classify_P300_bi.py

```diff
@@ -0,0 +1,144 @@
+"""
+====================================================================
+ERP EEG decoding with Quantum Classifier.
```

Comment:
You could change the title, as this is the same title than `plot_classify_EEG_quantum.py` and they should appears dfferently on the doc.